### DISCUSSION TITLE
反応しないキー　赤枠

### DISCUSSION THREAD

**Discussion by 10susumu** [Yowkees/keyball#660] [Q&A]
![基盤写真](https://github.com/user-attachments/assets/dbbe2308-5a4c-4fd2-9256-284bd6dce9d1)

初心者の質問で申し訳ありません
赤枠で囲った部分で反応しないキーがありダイオードのハンダ不良と考えたのですが、
どのダイオードをハンダし直せばよいでしょうか？
宜しくお願い致します

**Comment by 10susumu** [comment]
PCBのデータを確認しました
赤枠から２つ下のダイオードのハンダ付けをやり直して見ます
![回路図トリミング](https://github.com/user-attachments/assets/6a9c028d-d4ef-40ea-913c-f48e3863fb47)

**Comment by Yowkees** [comment]
ご回答できておらず失礼しました。

２つ下のダイオードで合っています。写真を拡大してみると、確かに左側の足がテンプラしているように見えますので、こちらを再半田してみてください。

お手数ですが、ご対応お願いいたします。
